tests/Cache/CacheClientTest.php

```diff
@@ -3297,6 +3297,59 @@ public function testSortedSetFetchByRankWithNegativeStartRankLessThanEndRank_Thr
         $this->assertEquals(MomentoErrorCode::INVALID_ARGUMENT_ERROR, $response->asError()->errorCode());
     }
 
+    public function testSortedSetRemoveElement_HappyPath()
```

Comment:
TODO: also test a sorted set with multiple elements:
- remove an element in the sorted set; the sorted set is still there but lacking that element
- remove an element not in the sorted set; the sorted set is the same as before